### ISSUE TITLE
PSA: Haunted Forest will be under "Story" activity mode, and will have no new stats

### ISSUE THREAD

**Issue by vthornheart-bng** [Bungie-net/api#725]
Due to internal time constraints and risks given that timeline, the Haunted Forest will not have its own activity mode, nor will we be able to add one retroactively to the data.  I heard that some people were thinking of building stats-related features around the activities, and so I wanted to give you all a heads up as far in advance as possible.  It's a bummer, and something I'm hoping we will avoid in the future: but at least for this time around, we're out of luck on providing the haunted forest its own specific stats and mode grouping.

**Comment by ckhicks** [comment]
Thanks for the heads up!

**Comment by kramrm** [comment]
Looks like the Verdant Forest is in the same situation.  I understand these are limited time events, but having some unique stats to track would be nice to provide extra incentive to promote (friendly) competition and encourage more event participation.

**Comment by vthornheart-bng** [comment]
I agree with that sentiment wholeheartedly, and have brought the verdant forest situation to the attention of the teams working on it.  Hopefully next time around we can provide a better experience from a stats standpoint!

**Comment by kramrm** [comment]
Thanks! 🤞
